docs/submission/faq.md

```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
 ---
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
+
+## Frequently Asked Questions
+
+1. How do I submit multiple values for a field? For example, I want to submit multiple symptoms for the `presenting_symptoms` field.
+
+   To include multiple values, separate values with a pipe delimiter '|' within your file. For example: `Anemia|Fatigue|Fever`.
```

Comment:
Do we need to say something like "when permitted"? not all the fields allow for multiple values.  Like we can say, the fields that allow for multiple values will have these instructions specified in the notes section of the dictionary.

Reply:
Yes, I updated this sentence so it's clear now.